web/public/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
```

Comment:
it is a good practice to have a `<meta name="description" content="foo" />` at a minimum in the head. I'd recommend using a library like `react-helmet` to achieve this